pages/common/krunvm.md

```diff
@@ -0,0 +1,24 @@
+# krunvm
+
+> CLI-based utility for creating microVMs from OCI images.
+> More information: <https://github.com/containers/krunvm>.
+
+- Create MicroVM based on Fedora:
```

Comment:
We try to avoid abbreviations. what is a µVM? and which is the difference with a classic VM?

Reply:
There is plenty of info about this check this out https://williamdurand.fr/2022/07/11/containers-and-micro-virtual-machines/

Reply:
ok, it's a new "concept" and doesn't have a page in Wikipedia.